app/views/admin/groups/_users.haml

```diff
@@ -1,17 +1,32 @@
+= simple_form_for :group_users, url: admin_group_users_path(@group) do |f|
+  .d-flex
+    .ml-auto.w-50
+      = f.input :user_ids, collection: User.where.not(id: @group.user_ids), as: :select_two, input_html: { multiple: true }, label: 'Users to Add'
+    .ml-4
+      = f.button :submit, class: 'btn btn-secondary mt-no-label' do
+        %i.icon-plus
+        Add Users
+
 - if @group.users.count.positive?
-  %h3= pluralize(@group.users.count, 'User')
+  %h3.mt-4= pluralize(@group.users.count, 'User')
```

Comment:
let's move the H3 out of the conditional clause.

Reply:
Done